src/oracles/ChainlinkPriceOracle.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {AggregatorV3Interface} from "../interfaces/AggregatorV3Interface.sol";
+import {IPriceOracle} from "../interfaces/IPriceOracle.sol";
+import {IWatchtowerCustomErrors} from "../interfaces/IWatchtowerCustomErrors.sol";
+/**
+ * @title Chainlink Price Oracle
+ * @author GUNBOATs
+ * @dev This contract creates an oracle that is compatible with the IPriceOracle
+ * interface and can be used by a CoW AMM to determine the current price of the
+ * traded tokens from Chainlink Data Feeds with decimals lower than 18.
+ */
+
```

Comment:
nit: doc comments should have have no new line between the actual function. Applies to multiple cases in this PR.